hello-strongbox-pypi/.pypirc.template

```diff
@@ -0,0 +1,14 @@
+[distutils]
+index-servers=
+  strongbox
+  testpypi
+
+[strongbox]
+repository: http://localhost:48080/storages/storage-pypi/pypi-releases
+username: admin
+password: password
+
+[testpypi]
```

Comment:
I don't think we need this for the sake of the example.

Reply:
Fixed.